documentation/release_3.1.htm

```diff
@@ -225,6 +226,7 @@ <h3>Changed functionality</h3>
     trigger or frame times in a series is no longer enforced. This allows 3-D DICOM volumes to be read correctly but may cause
     issues for dynamic PET or CT data.
   </li>
+  <li> As mentioned in the breaking of backwards compatibility, images must be set prior to using the forward and back projectors. To tell the back projector to start projecting into a new image, <tt>reset_output</tt> must be used. The result of the back projection is then returned via <tt>get_output</tt>. This was done so that using data processors (such as motion or smoothing) prior to the forwards projection or following the back projection will be more efficient.</li>
```

Comment:
add something about `set_input` for the forward projector as well. Also, I'd prefer to have some line breaks (don't matter for html, but easier to read on github!)